maint-lib/flavors-modified-vs-master.py

```diff
@@ -42,9 +42,10 @@ def _filediff_intersects_sources(filediff, sources):
 
 # Get list of files different from VS_BRANCH
 try:
+    _run_cmd(['git', 'fetch', 'origin', 'master'])
```

Comment:
This command outputs the below text  before the flavor output.
```
From github.com:SUSE/ceph-container
 * branch            master     -> FETCH_HEAD
```
Is this intentional? It seems like a side-effect. I originally wrote this to only return flavors modified with no other text so it could be used as input for scripting.

Reply:
You're right, this actually breaks one of my scripts. At first, I thought this was a good indication. Let me fix that.

Reply:
Fix in https://github.com/ceph/ceph-container/pull/976